install_scripts/templates/common/kubernetes-upgrade.sh

```diff
@@ -134,6 +136,57 @@ maybeUpgradeKubernetes() {
 
     upgradeK8sRemoteMasters "1.13.5" "$K8S_UPGRADE_PATCH_VERSION"
     upgradeK8sWorkers "1.13.5" "$K8S_UPGRADE_PATCH_VERSION"
+
+    if [ "$KUBERNETES_TARGET_VERSION_MINOR" -eq "13" ]; then
+        return 0
```

Comment:
do yuo have to `rm /opt/replicated/upgrade` in all these return cases?

Reply:
yeah, I guess I would